src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
```

Comment:
if u don't need to change object getting data one by one, u already have all what u need to fill this object - use constructor with parameters. And write simpler code
(and don't forget to think about picking color method in enum)

```suggestion
    public Ball getRandomBall() {
        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
    }
```